packages/clerk-js/src/core/clerk.ts

```diff
@@ -1654,6 +1672,16 @@ export class Clerk implements ClerkInterface {
     this.environment = environment;
   }
 
+  public prefetchDependencies = async (): Promise<void> => {
+    if (this.isCoinbaseWalletEnabled) {
```

Comment:
As an optimization we should add the follwing

`if(signedIn && singleMode) then don't load the sdk`